src/main/java/com/ekart/model/User.java

```diff
@@ -0,0 +1,74 @@
+package com.ekart.model;
+
+import com.vladmihalcea.hibernate.type.json.JsonType;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+@TypeDefs({
+        @TypeDef(
+                name = "json",
+                typeClass = JsonType.class
+        )
+})
+public class User {
+    @Id
+    @SequenceGenerator(
+            name = "user_sequence",
+            sequenceName = "user_sequence"
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_sequence"
+    )
+    private Long id;
+
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+    private byte[] password;
+
+    private String email;
+
+    @Type(type = "json")
+    @Column(columnDefinition = "jsonb")
+    private List<Address> addresses;
```

Comment:
it's better to maintain new table for Address with onetomany mapping. In this approach it will not be possible to edit, delete or add new address